tracker/peerstore/redis.go

```diff
@@ -19,11 +19,11 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/andres-erbsen/clock"
+	"github.com/garyburd/redigo/redis"
```

Comment:
Can you add a space and put external dependencies below internal dependencies?

Reply:
Done.